### ISSUE TITLE
`suspend()` and `resume()` not implemented for `OfflineAudioContext`

### ISSUE THREAD

**Issue by UlyssesZh** [ircam-ismm/node-web-audio-api#30]
```js
import { OfflineAudioContext } from 'node-web-audio-api';
const context = new OfflineAudioContext(2, 44100, 44100);
const promise = context.startRendering();
await context.suspend(0.1); // Uncaught TypeError: context.suspend is not a function
await context.resume();
const audioBuffer = await promise;
```
In [this issue comment](https://github.com/ircam-ismm/node-web-audio-api/issues/4#issuecomment-1289129580), @b-ma said that they were deleting `suspend()` and `resume()` for `OfflineAudioContext`:

> it should be mostly duplicating the file, modifying the constructor to handle the constructor signature and removing the extra online context methods (resume, suspend, etc.)

However, they are actually part of `OfflineAudioContext`, **not exclusive** to the online `AudioContext`, [according to the WAA specs](https://webaudio.github.io/web-audio-api/#dom-offlineaudiocontext-resume).

**Comment by b-ma** [comment]
Hey,

Yup, that was actually a mis-wording from my part, indeed these methods exists in `OfflineAudioContext` but they are not yet implemented in the upstream library so the bindings do not exists yet.

**Comment by UlyssesZh** [comment]
Oh, I see. I just noticed orottier/web-audio-api-rs#100.

**Comment by orottier** [comment]
Fixed in #50 (might need a new release?)
